capture_image.py

```diff
@@ -12,18 +12,30 @@
 # finds rotation and translation of vision targets
 class CaptureImage:
 
-    # initilaze variables
-    def __init__(self, detector):
+    # # initilaze variables
+    # def __init__(self, detector):
```

Comment:
Do you still need this code? If not, delete it. You can see the history in Git, no need to keep old code commented.